cmd/broker/main_suite_test.go

```diff
@@ -90,9 +90,10 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
```

Comment:
Can we add something in the suite test to make sure the right node packages are installed? Or add that to the makefile or something?

Reply:
thats a good idea. will look into it.